### ISSUE TITLE
Improving UX around transactions sent and received

### ISSUE THREAD

**Issue by akhi3030** [near/near-explorer#1042]
Hi team.  

In the accounts view on the explorer, the transactions tab feels a bit obtuse to me.

In the attached picture, it is not clear to me by looking at the arrows which number represents sent and which number represents received. I don't know if others also find it unclear. Do you think that we should consider using a different methodology to indicate what the two numbers represent?

<img width="382" alt="transaction-view" src="https://user-images.githubusercontent.com/4679209/173025369-789fb89f-4af2-4fd9-b480-b0b0e40f396a.png">

**Comment by frol** [comment]
@reefoh @luixo it is not the first time we hear that the arrows are not intuitive.

@akhi3030 we are working on a redesigned representation of the account details page. Here is an early preview: https://frontend-pr-917-gum7.onrender.com/beta/accounts/frol.near (feel free to replace the account id in the URL with the account you care). How do you like it?

**Comment by frol** [comment]
> Is that correct?

That is correct. So far nobody brought the need of differentiation to us and we might consider showing the breakdown on hover if the request will ever reach us.

> Being fairly new to the project, I am not sure how important that differentiation is. So this can be just fine as well.

Thanks for sharing, I guess this speaks for itself. If newcomers don’t care, we should not throw unnecessary information at them

**Comment by akhi3030** [comment]
Always happy to not have to parse info that I do not need 😆
